.semaphore/semaphore.yml.d/blocks/20-felix.yml

```diff
@@ -247,7 +247,47 @@
         - mkdir artifacts
         - ./.semaphore/create-test-vms ${VM_PREFIX}
     jobs:
-      - name: UT/FV tests on new kernel
+      - name: BPF UT/FV tests
+        env_vars:
+          - name: FELIX_FV_NFTABLES
+            value: "Enabled"
+          - name: FELIX_FV_BPFATTACHTYPE
+            value: "tc"
+        execution_time_limit:
+          minutes: 180
+        commands:
+          - ./.semaphore/run-tests-on-vms ${VM_PREFIX}
+    epilogue:
+      always:
+        commands:
+          - ./.semaphore/collect-artifacts-from-vms ${VM_PREFIX}
+          - ./.semaphore/publish-artifacts
+          - ./.semaphore/clean-up-vms ${VM_PREFIX}
+    secrets:
+      - name: google-service-account-for-gce
+- name: "Felix: BPF UT tests on Ubuntu 25.04"
+  run:
+    when: "${FORCE_RUN} or change_in(['/*', '/api/', '/libcalico-go/', '/typha/', '/felix/', '/cni-plugin/pkg/dataplane/linux/dataplane_linux.go'], {exclude: ['/**/.gitignore', '/**/README.md', '/**/LICENSE']})"
+  dependencies:
+    - Prerequisites
+  task:
+    prologue:
+      commands:
+        - cd felix
+        - export GOOGLE_APPLICATION_CREDENTIALS=$HOME/secrets/secret.google-service-account-key.json
+        - export SHORT_WORKFLOW_ID=$(echo ${SEMAPHORE_WORKFLOW_ID} | sha256sum | cut -c -8)
+        - export ZONE=europe-west3-c
+        - export VM_PREFIX=sem-${SEMAPHORE_PROJECT_NAME}-${SHORT_WORKFLOW_ID}-felix-2504-
+        - echo VM_PREFIX=${VM_PREFIX}
+        - export REPO_NAME=$(basename $(pwd))
+        - export IMAGE=ubuntu-2504-plucky-amd64-v20250805
+        - export NUM_FV_BATCHES=0
+        - export RUN_UT=true
+        - export FV_FOCUS='_BPF_.*ct=true'
```

Comment:
Can you please export the ```UBUNTU_VERSION``` and ```DOCKER_VERSION``` similar to https://github.com/projectcalico/calico/blob/master/.semaphore/semaphore.yml.d/blocks/20-felix.yml#L211